src/main.rs

```diff
@@ -43,11 +45,17 @@ fn main() {
         (_, _) => prompt(&mut src),
     };
 
+
     match result {
         Ok(_game) => {
             src.print("And the games begin!"); // Do something with the games here.
-                                               //Call the interpreter
-                                               //derpreter(game);
+            println!("Where would you like to go?"); //Call the interpreter. derpreter(game);
```

Comment:
Let's move this block out of main into `evaluate.rs` or something. Ideally we want to put as little logic as possible in this file, and leave most of it within the lib.